frontend/src/lib/components/ServiceRequestPreview.svelte

```diff
@@ -0,0 +1,107 @@
+<script lang='ts'>
+	import messages from '$media/messages.json'
+	import { goto } from '$app/navigation';
+	import { Badge, Card } from 'stwui';
+	import { Dropdown } from 'stwui';
+	import type { GetServiceRequestsResponse, ServiceRequest } from '$lib/services/Libre311/Libre311';
+	import dropDownIcon from '$lib/assets/ellipsis-vertical.svg';
+
+	export let serviceRequest: GetServiceRequestsResponse;
```

Comment:
`export let serviceRequest: GetServiceRequestsResponse;`
change the type to be:
`export let serviceRequest: ServiceRequest;`

The former type denotes is a list of service requests.